gammapy/spectrum/reflected.py

```diff
@@ -414,3 +419,124 @@ def plot(self, fig=None, ax=None, cmap=None, idx=None, add_legend=False):
             ax.legend(handles=handles)
 
         return fig, ax, cbar
+
+
+class ReflectedRegionsBackgroundMaker:
+    """Reflected regions background maker.
+
+    Parameters
+    ----------
+    region: `~regions.SkyRegion`
+        On region to compute off regions for.
+    angle_increment : `~astropy.coordinates.Angle`, optional
+        Rotation angle applied when a region falls in an excluded region.
+    min_distance : `~astropy.coordinates.Angle`, optional
+        Minimal distance between two consecutive reflected regions
+    min_distance_input : `~astropy.coordinates.Angle`, optional
+        Minimal distance from input region
+    max_region_number : int, optional
+        Maximum number of regions to use
+    exclusion_mask : `~gammapy.maps.WcsNDMap`, optional
+        Exclusion mask
+    binsz : `~astropy.coordinates.Angle`
+        Bin size of the reference map used for region finding. Default : 0.01 deg
```

Comment:
Suggest to remove the "Default: 0.01 deg" here and almost everywhere in Gammapy.
It's very hard to maintain, we have maybe ~ 10,000 parameters in function calls with defaults, and they often change and people forget to edit the docstring.

Reply:
Agreed. This was a copy and paste error from `ReflectedRegionsFinder`, I modified the other docstrings in `reflected.py` as well.